src/pages/workspace/accounting/netsuite/advanced/NetSuiteAdvancedPage.tsx

```diff
@@ -0,0 +1,286 @@
+        },
+        {
+            type: 'divider',
+            key: 'divider5',
+        },
+        {
+            type: 'toggle',
+            title: translate('workspace.netsuite.advancedConfig.customFormID'),
+            subtitle: translate('workspace.netsuite.advancedConfig.customFormIDDescription'),
+            isActive: !!config?.customFormIDOptions?.enabled,
+            switchAccessibilityLabel: translate('workspace.netsuite.advancedConfig.customFormIDDescription'),
+            shouldPlaceSubtitleBelowSwitch: true,
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+            onToggle: (isEnabled) => Connections.updateNetSuiteCustomFormIDOptionsEnabled(policyID, isEnabled),
+            pendingAction: config?.pendingFields?.customFormIDOptions,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+        },
+        {
+            type: 'menuitem',
+            description: translate('workspace.netsuite.advancedConfig.customFormIDReimbursable'),
+            onPress: () => {}, // TODO: This will be implemented in a future PR
+            brickRoadIndicator: config?.errorFields?.customFormIDOptions ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: config?.customFormIDOptions?.reimbursable[CONST.NETSUITE_MAP_EXPORT_DESTINATION[config.reimbursableExpensesExportDestination]],
+            pendingAction: config?.pendingFields?.customFormIDOptions,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+            shouldHide: !config?.customFormIDOptions?.enabled,
+        },
+        {
+            type: 'menuitem',
+            description: translate('workspace.netsuite.advancedConfig.customFormIDNonReimbursable'),
+            onPress: () => {}, // TODO: This will be implemented in a future PR
+            brickRoadIndicator: config?.errorFields?.customFormIDOptions ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: config?.customFormIDOptions?.reimbursable[CONST.NETSUITE_MAP_EXPORT_DESTINATION[config.nonreimbursableExpensesExportDestination]],
+            pendingAction: config?.pendingFields?.customFormIDOptions,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.CUSTOM_FORM_ID_OPTIONS),
+            shouldHide: !config?.customFormIDOptions?.enabled,
+        },
+    ];
+
+    return (
+        <ConnectionLayout
+            displayName={NetSuiteAdvancedPage.displayName}
+            headerTitle="workspace.accounting.advanced"
+            headerSubtitle={config?.subsidiary ?? ''}
```

Comment:
with long page content, we should add the padding bottom to prevent content is cut off at the bottom.

More details: https://github.com/Expensify/App/issues/55238#issuecomment-2591053852